### PR TITLE
Adds more tests to manifest; fixes issues raised by tests.

### PR DESCRIPTION
**Pull request by paullewis**

### CODE REVIEW

File: audits/manifest/theme-color.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ class ManifestThemeColor {
     let hasThemeColor = false;
     const manifest = inputs.manifest;
 
-    if (manifest) {
+    if (manifest && manifest.theme_color) {
```
Comment (addyosmani) [inline audits/manifest/theme-color.js]: :+1:

File: audits/manifest/icons-192.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ class ManifestIcons192 {
     let hasIcons = false;
     const manifest = inputs.manifest;
 
-    if (manifest && manifest.icons.value) {
+    if (manifest && manifest.icons && manifest.icons.value) {
```
Comment (addyosmani) [inline audits/manifest/icons-192.js]: :+1:

**Comment by addyosmani** [comment]
LGTM. I have some suggestions for improvement, but per talking to you in person will file an issue :)

**Comment by paullewis** [comment]
Cheers dude! :checkered_flag:

File: test/audits/manifest/icons-192.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  it('fails when a manifest contains no icons', () => {
+    const inputs = {
+      manifest: {
+        icons: null
+      }
+    };
+
+    return assert.equal(Audit.audit(inputs).value, false);
+  });
+
```
Comment (brendankenny) [inline test/audits/manifest/icons-192.js]: if we want to be super thorough, we should probably add a test for an icon without a size and an icon with multiple sizes and 192x192 is in there somewhere. I can add those really fast

**Comment by paullewis** [comment]
Add it for sure. The more robust the happier.

**Comment by brendankenny** [comment]
this is great

File: test/audits/manifest/icons-192.js (hunk 3 above)
Comment (samccone) [inline test/audits/manifest/icons-192.js]: :+1:

**Pull request merged**